protobuf/execapi/api.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+import "protobuf/definition/execution.proto";
+
+package api;
+option go_package = "github.com/mesg-foundation/core/protobuf/execapi";
```

Comment:
Could you move this file to `protobuf/api/execution.proto` like anthony suggest in https://forum.mesg.com/t/execution-db-and-api/280

Reply:
The file is still name api.proto. it should be execution.proto

Reply:
Conversation moved to https://github.com/mesg-foundation/engine/issues/1013